### DISCUSSION TITLE
Cannot access the website on school wifi

### DISCUSSION THREAD

**Discussion by JiangZehua** [alshedivat/al-folio#2276] [Q&A] [answered]
Hi! I pulled the latest commit from master branch, and deploy the website on github as the instruction suggested. But looks like after the website is successfully deployed, it cannot be accessed from my university wifi, but can be access from my phone hotspot. 

Thank you in advance!

**Comment by george-gca** [comment]
Check if any of the other [sites that were created from our template](https://github.com/alshedivat/al-folio?tab=readme-ov-file#user-community) and that are also hosted on Github pages work. If they don't, it is your university that is blocking the access and there is nothing we could do about it.

**Reply by JiangZehua** [reply]
Thanks for you reply!

I double checked that if I use a older version, I can access it from our university network. The "older" version is the website I built for my friend from the commit on March 2023. I used that version of template to build her website in Jan 2024, and didn't update since then. It's working well.

**Reply by george-gca** [reply]
If that is the case, I would suggest you do a test to check if a specific library is being blocked. The easiest way to do this would be to get the latest version and comment out all the libraries that differ from her site, and check if that works. If it works, add libraries one by one until you find the culprit.

Just out of curiosity, in which university/country are you in?

**Reply by JiangZehua** [reply]
Thanks! I'll take a look. Sorry I was busy with other things until now.

I'm in NYU in the US. I found that it's not working in particularly when using google chrome as browser and using the school wifi (Eduroam). Safari looks fine. I'm not 100% sure since now I'm traveling and cannot reproduce the error.

**Comment by george-gca** [comment, marked as answer]
One possible solution to this now is setting `download` to true in the [third_party_libraries part of _config.yml](https://github.com/alshedivat/al-folio/blob/36bc24f7e4f8525ce6f00ccb234073771c898430/_config.yml#L422), as added in [08d562](https://github.com/alshedivat/al-folio/commit/08d562a1048e079b28f3fd6eed6ab0f6e35f5adb).

**Reply by geunee20** [reply]
Hello,

I recently deployed my webpage at https://geunee20.github.io/, but I'm encountering an issue with accessing it through **Eduroam**. For context, Eduroam is a network service providing internet access across various campuses and institutions worldwide, which normally allows seamless connectivity. However, while I can access other pages without any issues, mine seems to be blocked or unavailable. I set "download" to true but still does not work.

**Reply by george-gca** [reply]
Can you access the template demo?

My experience with eduroam is actually really bad. Laggy connection, if working at all, to access some sites unrelated to university websites. I only used when I wanted to access specific sites as if I was inside the university campus.

**Reply by geunee20** [reply]
I was able to access the template demo at https://alshedivat.github.io/al-folio/, which led me to suspect that the issue might be due to my webpage not being registered with Eduroam's DNS. However, it's been over a week since the webpage was created.

**Reply by george-gca** [reply]
Are you using the same libraries as the original repo? If you are, honestly idk if there's much you can do.
